### DISCUSSION TITLE
Some vector data are merged with raster reference map while Qfield project is used on field

### DISCUSSION THREAD

**Discussion by javrain** [opengisch/QField#2212] [Q&A]
We create a QField project with QFieldSync. The structure of the content of this project changed when QField project was used on a PAD and came back from field. A part of some vector layer features are lost as vector features, but are still visible as rasterised data, encapsulated into the raster reference map.

### Reproduction Steps:

1. When we send data to Qfield (Qfield project built with QFieldSync), some data are loaded as vector. Orthophoto is also loaded in same QField project (ECW tiles). Some vector layers are modifiable, some are not. _**(See SentToField.JPG)**_
2. On the return from field, same Qfield project contains same data, but some vector layers have been merged with orthophoto reference map. More specifically, vector layers are still in the QField project, but empty, while their objects have been inserted into orthophoto as raster data. Merged layers were "not modifiable" layers. _**(See ReturnFromField.JPG)**_

### Expected behavior
Having same vector layers with same features in the returned project, as it was in the initial one.

### Observed behavior:
Some vector layers have been merged with orthophoto reference map. In _**ReturnFromField.JPG**_, you can see the "lot" boundaries that are encapsulated raster reference map (overlined in yellow)

### Screenshots and GIFs
2 screenshot before and after field operations
Before
![SentToField](https://user-images.githubusercontent.com/92161537/136557279-588e8def-3e53-473d-881e-fd57b0b035d6.JPG)
After
![ReturnFromField](https://user-images.githubusercontent.com/92161537/136557251-1c9927da-6ada-4619-a62a-4610ee818a2b.JPG)

2 zips with data and QField project before and after field operations
Before
[Benga-Lot08_Before.zip](https://github.com/opengisch/QField/files/7311366/Benga-Lot08_Before.zip)

After
[Benga-Lot08.zip](https://github.com/opengisch/QField/files/7311341/Benga-Lot08.zip)

## Versions:
QField version: 1.9.6 (d5dd01 v1.9.6)
QField Sync version: 3.4.3

## Additional information:
* Problem started happening recently, didn't happen in an older version of QField: Don't know
* Problem can be reliably reproduced, doesn't happen randomly: Yes
* Problem happens with all files and projects, not only some files or projects: Yes

**Comment by nirvn** [comment]
@javrain , can you take a photo of the QField project properties panel? Looks like you might have setup some layers to rasterize when exporting?

**Reply by javrain** [reply]
"Looks like you might have setup some layers to rasterize when exporting": **CORRECT**
Thanks for your relevant answer.

Therefore, this discussion can be closed.
